pkg/azuredisk/azuredisk_v2.go

```diff
@@ -475,7 +481,7 @@ func (d *DriverV2) registerAzDriverNodeOrDie(ctx context.Context) {
 		},
 	})
 
-	go azdiskInformer.Start(ctx.Done())
+	azdiskInformerFactory.Start(ctx.Done())
```

Comment:
We should utilize `azureutils.StartAndWaitForCacheSync` instead of manually starting the informer here.

Reply:
If we want to use `azureutils.StartAndWaitForCacheSync`, we need to create an object as GenericInformer interface. But this informer needs `Lister()` method [here](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/9356028355480bb971ea4bd811ae98b5d3498f2f/pkg/azuredisk/azuredisk_v2.go#L555), we need to add this method/field to GenericInformer interface and GenericAzInformerInfo struct that would cause all other informers need to initiate `lister` field which won't be used. That's why I manually start this informer. Any other suggestion?